Apps/Database/src/Migrations/20211206201847_RevertFKInUserPrefUserProfileChanges.cs

```diff
@@ -0,0 +1,40 @@
+// -------------------------------------------------------------------------
```

Comment:
This is not correct, I should not see RevertFKInUserPrefUserProfileChanges or RemovedForeignKeyUserPreferences.  I should see that AddFkInUserPreferenceTable is deleted and that GatewayDbContextModelSnapshot.cs is updated.  Finally, we should have Brians original migration added with the proper name.